javascript/node/selenium-webdriver/lib/webdriver.js

```diff
@@ -1177,9 +1177,15 @@ class Options {
       expiry = Math.floor(date.getTime() / 1000)
     }
 
```

Comment:
Based on the [related](https://w3c.github.io/webdriver/#dfn-cookie-same-site) webdriver specification
> Defaults to None if omitted when adding a cookie.

We should set sameSite cookie by default as "None" when omitted.

```suggestion
if (!sameSite) {
 sameSite = 'None'
}
```

Reply:
I'm not too sure we should do that given that: 
* no other Selenium binding is doing that: all seem to leverage on the underlying driver/browser implementation for the default value
* I think it's the Browsers' responsibility to define the default behavior. For instance, in Chrome 85 that's exactly what changed: [default is now](https://web.dev/samesite-cookies-explained/#changes-to-the-default-behavior-without-samesite) `Lax` for Chrome and soon for Firefox too. That change was the reason I opened this PR in the first place.
* with the new security proposal, `sameSite='None'` should always be accompanied by `Secure=true`, so we'd need to take that into account
* changing that would be a breaking change (nothing against it, but I'd like the maintainers' opinion)

Interested to hear thoughts from @AutomatedTester